src/provable/gen/transaction.ts

```diff
@@ -490,3 +511,51 @@ type AccountUpdate = {
 let AccountUpdate = provableFromLayout<AccountUpdate, Json.AccountUpdate>(
   jsLayout.AccountUpdate as any
 );
+
+type Account = {
```

Comment:
I think i've seen this type be define like 3 times, couldn't we export it once and re use it?

Reply:
It's auto-generated in three places :D Once in a "JSON" version, once in a "bigint" version (where field elements are bigints, for mina-signer) and once the "normal snarkyjs" version where field elements are `Field` instances
We could make it generic, but I think since it's auto-generated its fine (also easier to read than generic)

Reply:
okay true, thats fine :D thanks!